requirements.in

```diff
@@ -1,7 +1,7 @@
 basket-client
 beautifulsoup4
 boto3
-Django==3.0.14
+Django==3.1.11
```

Comment:
Some of our packages require Django < 3.2 still. We may need to wait until those are updated or determine which packages we can remove to get us to Django 3.2. But Wagtail 2.13 works fine with Django 3.1.*

Reply:
Just a heads up that we *may* hit some issues with Django 3.2 + wagtail-localize https://github.com/wagtail/wagtail-localize/issues/396

Reply:
Ahh yes Ive seen this with Django 3.2, but doesn't seem to affect Django 3.1. We'll want to set `DEFAULT_AUTO_FIELD` in the settings.py file when we upgrade to Django 3.2.